src/app/services/login.service.ts

```diff
@@ -37,4 +38,18 @@ export class LoginService {
     return this.http.delete(`${this._baseUrl}/posts/byPID/${postID}`) as Observable<readPost>;
   }
 
+  loggedIn(): string{
+    if(!!sessionStorage.getItem(Constants.session.user)){
```

Comment:
What is the point of doing `!!` wouldn't `if(sessionStorage.getItem(Constants.session.user))` work the same?